test/integration.js

```diff
@@ -605,4 +623,39 @@ describe('connect()', () => {
       done();
     }, 10);
   });
+
+  it('should build sparsed array', (done) => {
+    fetchMock
+      .get('http://localhost/turnip?limit=5&offset=5&q=dinner',
+        {
+          records: [
+            { 'id':'58e55786065039ceb9acb0e2', 'name':'Lucas' },
+            { 'id':'58e55786e2106a216fdb5629', 'name':'Kirkland' },
+            { 'id':'58e55786819013f1e810d28e', 'name':'Clarke' },
+          ],
+          total_records: 10
+        },
+        {
+          headers: { 'Content-Type': 'application/json' }
+        }).catch(503);
+
+    const store = createStore((state) => state,
+      { okapi: { url: 'http://localhost', tenant: 'tenantid' } },
+      applyMiddleware(thunk));
+
+    const Connected = connect(Sparsed, 'sparsed', mockedEpics, defaultLogger);
+    const inst = mount(<Root store={store} component={Connected} />);
+
+    setTimeout(() => {
```

Comment:
@mkuklis, could you please clarify why setTimeout is needed here?

Reply:
@ViktorSoroka07 this is taken from other tests in that file. You can try to remove it and see if things work as expected. At some point I think we ran into an issue where the `mount` was not completely done before `inst.find` could be used properly.